lib/src/characters.dart

```diff
@@ -49,9 +49,9 @@ abstract class Characters implements Iterable<String> {
   /// Returns [CharacterRange] positioned after the last character
   /// of this [Characters].
   ///
-  /// Allows iterating the characters of [string] backwards
-  /// using [CharacterRange.movePrevious],
-  /// as well as controlling the iteration in more detail.
+  /// Allows iterating the characters of [string] backwards using
+  /// [CharacterRange.moveBack], as well as controlling the iteration in more
```

Comment:
I'd still keep the line break after the comma, instead of splitting the following clause on two lines.

Reply:
Done